cmd/mailbox-controller/main.go

```diff
@@ -107,17 +95,14 @@ func main() {
 		logger.Error(err, "failed to create config from flags")
 		os.Exit(3)
 	}
-	edgeViewConfig, err := configForViewOfExport(ctx, espwRestConfig, "edge.kubestellar.io")
```

Comment:
The lines

```
	options := resolveroptions.NewOptions()
	espwRestConfig, err := options.EspwClientOpts.ToRESTConfig()
```

are a botch. See how those options are not made available on the command line?  Better to follow the pattern used for `rootClientOpts` and `rootRestConfig`.

Reply:
Changed